fmings/limeify-be
# Getsongs zc

## Description 
GET All Songs and GET Songs by Id

## Related Issue
#4 #5 
## Motivation and Context
This is so we are able to view all songs and also target individual songs

## How Can This Be Tested?

1. Fetching the branch down:getsongs-zc
2. checkout into that branch 
3. run code and swagger should load

## Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/cc9fb86d-fff0-4f95-bd97-a2ce4b0d338e)

## Types of changes

- [x ] New feature (non-breaking change which adds functionality)
